### DISCUSSION TITLE
Lazy loading components

### DISCUSSION THREAD

**Discussion by ekimkael** [unpoly/unpoly#600] [Q&A] [answered]
Hello guys please I wonder if it's possible to have 2 elements fetching data in lazy loading kind of way.

let's say I have this

```html
  <main>
    <section id="car-x-details"></section>
    <section id="similar-cars"></section>
  </main>
```
is there a way to have something like a fragment or a block that i can import who is just in charge of displaying `similar-cars`?

**Comment by davisums** [comment, marked as answer]
The discussion right before this one that you’ve posted covers exactly this: https://github.com/unpoly/unpoly/discussions/574

Right now you have to either write a custom compiler pre use up-poll. A native solution seems to be coming soon though!
